barnaba/enm.py

```diff
@@ -47,17 +47,20 @@ def __init__(self,pdb,sele_atoms=["C1\'","C2","P","CA","CB"],cutoff=0.9,sparse=F
         self.sparse=sparse
         cur_pdb = md.load_pdb(pdb)
         topology = cur_pdb.topology
-        if(sele_atoms=="AA"):
-            idxs = [atom.index for atom in topology.atoms if (atom.name[0] != "H" and not ( atom.name[0].isdigit() and atom.name[1] == "H") )]
+        if(sele_atoms=="AA" or sele_atoms==["AA"]):
+            idxs = topology.select("not type H")
```

Comment:
@giopina did you check that this gives the same result?

Reply:
It does for the two molecules used in the tests: sample1.pdb, a.k.a. add riboswitch, and 2n82.pdb (protein+RNA complex).

Reply:
OK good for me! 2N82 was indeed the case with arginines `NH1` and `NH2` that were creating troubles with the initial parser.

Reply:
I just double-checked on 1y26.pdb, 430d.pdb and SARCIN.pdb